server/sql/analysis/clck_view.sql

```diff
@@ -0,0 +1,22 @@
+--
+-- View for Intel(R) Cluster Checker
+--
+DROP VIEW IF EXISTS clck_1;
+CREATE OR REPLACE VIEW clck_1 AS
+    SELECT test_run.duration,
+        0 AS encoding,
+        test_run.exit_value AS exit_status,
+        submit.hostname,
+        ''::text AS node_names,
+        0 AS num_nodes,
+        test_run.test_run_command_id::character varying AS optionid,
+        test_run.ctid AS rowid,
```

Comment:
That is `test_run.ctid` I don't see that in the schema:
 * https://github.com/open-mpi/mtt/blob/master/server/sql/schemas-v3.sql